src/utils/httpClient.js

```diff
@@ -0,0 +1,6 @@
+import { BASE_URL } from '@/constans';
```

Comment:
There seems to be a typo in the import path. It should be '@/constants' instead of '@/constans'. This will likely cause an import error.